config/lib/seed.js

```diff
@@ -4,6 +4,11 @@ var mongoose = require('mongoose'),
   chalk = require('chalk'),
   crypto = require('crypto');
 
+// set default seed options
+var seedOptions = {
+  logResults: true // log the results of the seeding to the console
```

Comment:
That's a lot better, yes. It means we are not inventing another configuration option but instead re-using and existing one.